{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/password.ts

```diff
@@ -0,0 +1,23 @@
+import AlpineInstance, { AlpineComponent } from "alpinejs";
+
+const password = (...args: unknown[]): AlpineComponent => ({
```

Comment:
It might be worth making the args a ts interface/type rather than unknown[] given that args[0] is eventName and args[1] is value both here and in the input.ts with the optional args[2] for type. Then you also wouldn't have to do `as string`. Maybe that would mess up other things but seems like something worth throwing in quickly